department-of-veterans-affairs/va.gov-cms
# VACMS-9097 new facility help desk notification

## Description

Relates to #9097. Closed the old PR with the dirty history to keep the cases separate in 4 smaller PRs. Like was originally planned 😅 

- Sends out a template message using the newly created CMS Help Desk notifications user to support@va-gov.atlassian.net with a message about any new facility created so that the appropriate follow-up steps may happen. 
- Coordinated with Stefanie on the desired wording of the email template.
- Created view modes to display email subject/body correctly.

## Testing done
Created new facilities of the 6 types locally, verified that an email was sent in mailhog ( https://va-gov-cms.ddev.site:8026/ )

## Screenshots
Email generated from a new facility being created:
<img width="1135" alt="Screen Shot 2022-05-31 at 2 05 15 PM" src="https://user-images.githubusercontent.com/11279744/171284260-419ff76d-87c3-4b64-a9c4-1f3e27deced5.png">

I'm not sure it's set up 100% correctly (or if it is mailhog changing something somewhere), because the `from:` and `to:` fields look a little off to me... but it's using the same setup as both @swirtSJW's example and the example in the module 🤔

## QA steps

What needs to be checked to prove this works?  
- create a new facility (any of the 6 facility types: VAMC Facility, VBA Facility, NCA Facility, Vet Center, Vet Center Mobile Vet Center, or Vet Center Outstation) and save it. exact section and menu things don't really matter for this initial step.
- as an admin user: check `/admin/content/message` and see that a new message was created for the facility creation. I am not sure how to check mailhog on tugboat, but if you are testing locally, visit mailhog at https://va-gov-cms.ddev.site:8026/ and see a corresponding email captured.
<img width="1401" alt="Screen Shot 2022-05-31 at 2 11 27 PM" src="https://user-images.githubusercontent.com/11279744/171285389-03451074-cf48-47d3-961c-753a629d74b0.png">
